File: Java/Java Programs College/Java Asssignment/Fibonacci.java
import java.util.Scanner;
class Fibonacci
{
	public static void main(String args[])
	{
		int a=0,b=1,c,n,i;
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter the number of terms to be displayed");
		n=sc.nextInt();
		System.out.print(a+" "+b+" ");
		for(i=3;i<=n;i++)
		{
			c=a+b;
			System.out.print(c+" ");
			a=b;
			b=c;
		}

	} 
}

File: Python Programs/Fibonacci.py
n = int(input("Enter the number of terms "))
f1 = 0
f2 = 1
print(f1, end=" ")
for x in range(1, n):
    print(f2, end=" ")
    f3 = f1 + f2
    f1 = f2
    f2 = f3
